### DISCUSSION TITLE
Dbus

### DISCUSSION THREAD

**Discussion by jbbjarnason** [stephenberry/glaze#522] [Ideas]
I think the glaze reflection could be reused greatly to seriallise and deserialise dbus binary format. Would it be an acceptable feature in glaze?

https://dbus.freedesktop.org/doc/dbus-specification.html

**Comment by stephenberry** [comment]
I’m happy to support other formats in Glaze as long as I’m not the primary bug fixer and person handling feature requests (for that spec). I do think it makes sense to use the same API for various formats like Dbus.
I’m just currently focused on what I’m using, which is JSON, CSV, and BEVE. But, I will likely add more in the future.
If you want to drive this development, I would support it.

**Reply by jbbjarnason** [reply]
Great to hear, me and my colleague @omarhogni will drive the development and contribute to supporting dbus format in the future in glaze repo.

**Reply by stephenberry** [reply]
Fantastic!

**Reply by stephenberry** [reply]
Should I add @omarhogni as a collaborator to Glaze?

**Reply by omarhogni** [reply]
I think there is no need for now well open a pull request when we have something worth looking at :+1:
